cluster/src/main/java/org/apache/iotdb/cluster/config/ClusterConfig.java

```diff
@@ -127,6 +125,34 @@
    */
   private int readDataConsistencyLevel = 1;
 
+  /**
+   * Max num of threads can concurrently execute qp sub-tasks of all client requests. When <= 0, use
+   * CPU core number * 10. Each client request corresponds to a QP Task. A QP task may be divided
+   * into several sub-tasks.
+   */
+  private int concurrentQPSubTaskThread = Runtime.getRuntime().availableProcessors() * 10;
+
+  /**
+   * Max num of threads can concurrently apply tasks in data partition state machine. When <= 0, use
+   * CPU core number.Raft Task refers to tasks that need to be performed in the state machine.
+   */
+  private int concurrentRaftTaskThread = Runtime.getRuntime().availableProcessors();
```

Comment:
For each raft log, its state machine will start a thread to apply this log. This value indicates maximum number of threads for a state machine to create. Currently, this param only works for state machine in data partition holder. Staten machine in metadata holder will apply raft log serially.
When this value <= 0, use CPU core number.

Reply:
I prefer "Currently, this param only works for state machine of leader in data partition holder."

Reply:
This param is useless now?